pom.xml

```diff
@@ -222,6 +221,10 @@
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.8.0</version>
+                    <configuration>
+                        <source>1.8</source>
+                        <target>1.8</target>
+                    </configuration>
```

Comment:
why Java 8 instead of 17 like any other MS?

Reply:
i have a problem with dependency in aws, build failure